CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # FL Accelerator Changelog
 
+##  November 2022 release
+
+We are excited to announce the release of the November iteration of our [FL Accelerator repository](https://github.com/Azure-Samples/azure-ml-federated-learning).
+
+Here are the new features.
+
+<!-- ### Provisioning
+-  -->
+
+### Documentation
+- We have introduced a new [Troubleshooting Guide](./docs/tsg.md) to help you troubleshoot common issues. We will keep adding to it as we become aware of more common issues.
+
+### FL Experience
```

Comment:
could we move this up in the release notes? or maybe mention on the top that the main value add of this release are the 3 examples?

Reply:
done